examples/dialogflow-entities-example/main.py

```diff
@@ -0,0 +1,226 @@
+    #   https://github.com/googleapis/googleapis/blob/master/google/cloud/dialogflow/v2/entity_type.proto#L397
+    #   https://dialogflow-python-client-v2.readthedocs.io/en/latest/_modules/dialogflow_v2/gapic/entity_types_client.html#EntityTypesClient.batch_update_entity_types
+
+    entities_batch = {
+        "entity_types": [{
+            "name":
+                "projects/<project_id>/agent/entityTypes/51637fc0-f717-4458-aef1-3d6a51d113f5",
+            "display_name":
+                "saving-account-types",
+            "kind":
+                dialogflow_v2.entity_types_client.enums.EntityType.Kind.
+                KIND_MAP,
+            "entities": [{
+                "value":
+                    "saving-account-types",
+                "synonyms": [
+                    "saving", "saving account", "child saving", "IRA", "CD",
+                    "student saving"
+                ]
+            }]
+        }, {
+            "name":
+                "projects/<project_id>/agent/entityTypes/6e4490bc-ec73-468a-be1a-264910421155",
+            "display_name":
+                "checking-account-types",
+            "kind":
+                dialogflow_v2.entity_types_client.enums.EntityType.Kind.
+                KIND_MAP,
+            "entities": [{
+                "value":
+                    "checking-account-types",
+                "synonyms": [
+                    "checking", "checking account", "student checking account",
+                    "student account", "business checking account",
+                    "business account"
+                ]
+            }]
+        }, {
+            "display_name":
+                "account_types",
+            "kind":
+                dialogflow_v2.entity_types_client.enums.EntityType.Kind.
+                KIND_LIST,
+            "entities": [{
+                "value": "@saving-account-types:saving-account-types",
+                "synonyms": ["@saving-account-types:saving-account-types"]
+            }, {
+                "value": "@checking-account-types:checking-account-types",
+                "synonyms": ["@checking-account-types:checking-account-types"]
+            }, {
+                "value":
+                    "@sys.date-period:date-period "
+                    "@saving-account-types:saving-account-types",
+                "synonyms": [
+                    "@sys.date-period:date-period "
+                    "@saving-account-types:saving-account-types"
+                ]
+            }, {
+                "value":
+                    "@sys.date-period:date-period "
+                    "@checking-account-types:checking-account-types",
+                "synonyms": [
+                    "@sys.date-period:date-period "
+                    "@checking-account-types:checking-account-types"
+                ]
+            }]
+        }]
+    }
+
+    client = get_dialogflow_client()
+    parent = get_agent(client)
+
+    # Uncomment if you want to create entities one by one.
```

Comment:
It would be better to expose / document a parameter / flag for this than have commented out code like this.